### ISSUE TITLE
Functions list not rendering

### ISSUE THREAD

**Issue by obfuscurity** [graphite-project/graphite-web#1097]
The functions list is not rendering for `docs/functions.rst` in `0.9.x` or `0.9.13-pre1`.

/cc @gwaldo

**Comment by obfuscurity** [comment]
From the latest build:

```
Could not import graphite.local_settings, using defaults!
/var/build/user_builds/graphite/checkouts/0.9.13-pre1/webapp/graphite/settings.py:244: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
  warn('SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security')
/var/build/user_builds/graphite/checkouts/0.9.13-pre1/docs/functions.rst:17: WARNING: autodoc: failed to import module u'graphite.render.functions'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/graphite/envs/0.9.13-pre1/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 335, in import_object
    __import__(self.modname)
  File "/var/build/user_builds/graphite/checkouts/0.9.13-pre1/webapp/graphite/render/functions.py", line 24, in <module>
    from graphite.render.datalib import TimeSeries
  File "/var/build/user_builds/graphite/checkouts/0.9.13-pre1/webapp/graphite/render/datalib.py", line 22, in <module>
    from graphite.storage import STORE, LOCAL_STORE
  File "/var/build/user_builds/graphite/checkouts/0.9.13-pre1/webapp/graphite/storage.py", line 4, in <module>
    import whisper
ImportError: No module named whisper
/var/build/user_builds/graphite/checkouts/0.9.13-pre1/docs/releases/0_9_11.rst:2: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/var/build/user_builds/graphite/checkouts/0.9.13-pre1/docs/releases/0_9_11.rst:115: WARNING: Title underline too short.
```

**Comment by obfuscurity** [comment]
So it appears that the `.. automodule:: graphite.render.functions` in functions.rst is failing because the build has no permissions for `/opt/graphite` so it can't install the Whisper dependency for functions.py.

I'm a bit out of my element here with the RTD build environment. Anyone have clues?

/cc @mleinart @brutasse @drawks

**Comment by obfuscurity** [comment]
P.S. `0.9.12` is the last `0.9.x` tag with a working functions list. I hesitate to rebuild that one because it's probably going to fail for the same reason.

**Comment by gwaldo** [comment]
I'll take a look in a little while, but I've got nothing off of the top of my head.

**Comment by SEJeff** [comment]
It works in the 0.9.12 release and doesn't on the latest 0.9.23-pre1? I should be able to bust out some git bisect run later today and find/fix the culprit.

**Comment by obfuscurity** [comment]
Ok, I think I fixed this over at RTD. We use virtualenv for building docs, but `0.9.x` still has the hardcoded prefix problem. So I enabled the following setting and builds are passing now.

![screen shot 2015-01-11 at 10 17 41 am](https://cloud.githubusercontent.com/assets/494338/5695643/451e0ccc-997b-11e4-85b8-3e1602978c47.png)

I don't know if this is an actual fix or just a workaround but it WFM for now.
